### ISSUE TITLE
CogvideoX-5b adds freeze-frames in start and end of clip

### ISSUE THREAD

**Issue by tin2tin** [huggingface/diffusers#9413]
### Describe the bug

CogvideoX-5b adds 7 freeze-frames to the start and 1–2 freeze-frames to the end of the clip. 

Output file:
https://github.com/user-attachments/assets/7a471858-e6ed-423a-a8f4-74125f2a18ba

Looking at the output:
https://github.com/user-attachments/assets/568fba7c-2927-4b2a-8652-d69c27628d69

### Reproduction

Run this code:
```
import torch
from diffusers import CogVideoXPipeline
from diffusers.utils import export_to_video

# Models: "THUDM/CogVideoX-2b" or "THUDM/CogVideoX-2b"
pipe = CogVideoXPipeline.from_pretrained("THUDM/CogVideoX-2b", torch_dtype=torch.float16).to("cuda")
pipe.enable_model_cpu_offload()
prompt = (
    "Slow motion. A panda, dressed in a small, red jacket and a tiny hat, sits on a wooden stool in a serene bamboo forest. "
    "The panda's fluffy paws strum a miniature acoustic guitar, producing soft, melodic tunes. Nearby, a few other "
    "pandas gather, watching curiously and some clapping in rhythm. Sunlight filters through the tall bamboo, "
    "casting a gentle glow on the scene. The panda's face is expressive, showing concentration and joy as it plays. "
    "The background includes a small, flowing stream and vibrant green foliage, enhancing the peaceful and magical "
    "atmosphere of this unique musical performance."
)
video = pipe(prompt=prompt, guidance_scale=6, num_inference_steps=50, num_frames = 48).frames[0]
export_to_video(video, "C:/Users/peter/Documents/output.mp4", fps=8)
```

### System Info

Diffusers 0.30.2
Win 11
4090

### Who can help?

@DN6 @a-r-r-o-w

**Comment by a-r-r-o-w** [comment]
Marking as closed due to it being more of a training related problem, as this behaviour happens both with the original codebase as well as Diffusers randomly. The best solution, at the moment, for Cog-T2V models is to just roll the dice and hope your random seed works :(

Feel free to reopen if still applicable to diffusers-specific implementation in some way
